limacharlie/Hive.py

```diff
@@ -70,6 +70,8 @@ def set( self, record ):
             req[ 'etag' ] = record.etag
         if len( usrMtd ) != 0:
             req[ 'usr_mtd' ] = json.dumps( usrMtd )
+        if record.arl is not None:
+            req [ 'arl' ] = json.dumps(record.arl)
```

Comment:
This should always be a string right?
Not a big deal since json.dumps will produce a string from a string IIRC.